databricks/koalas/namespace.py

```diff
@@ -1576,6 +1576,8 @@ def get_dummies(
             "Length of 'prefix' ({}) did not match the length of "
             "the columns being encoded ({}).".format(len(prefix), len(column_labels))
         )
+    elif isinstance(prefix, dict):
+        prefix = [prefix[column_label[0]] for column_label in column_labels]
```

Comment:
Can you handle error cases such as `pd.get_dummies(pdf, prefix={"A": "foo"})`?

Reply:
Thank for the comment!

I think they are already handled in the existing code just same as pandas.

```python
>>> pd.get_dummies(pdf, prefix={"A": "bar"})
...
ValueError: Length of 'prefix' (1) did not match the length of the columns being encoded (2).

>>> ks.get_dummies(kdf, prefix={"A": "bar"})
...
ValueError: Length of 'prefix' (1) did not match the length of the columns being encoded (2).
```

Reply:
Oh, it shares the length check!

Reply:
Yup! :)